### DISCUSSION TITLE
Can this be added to dupeGuru PE?

### DISCUSSION THREAD

**Discussion by djblanch** [hsoft/dupeguru#1030] [Ideas]
I've been very glad to have dupeGuru to find duplicate picture files (even of smaller file sizes!). The one thing I would really like to see added is the ability to clear ALL of the folders in the search list at once, rather than having to select them one by one and press [-] each time. Even if it's not possible to add a "clear all" button (or menu item), it would be much faster if a new remaining folder was automatically chosen after each deletion, so that repeatedly pressing [-] would much more quickly clear the list. Other than this issue, I'm very happy with dupeGure, the way it works, and the confidence I can have that it will only show actual duplicates (based on my preference settings). Thanks to all who have contributed to making this great app available!!

D.B.
Gresham, OR, USA

**Comment by arsenetar** [comment]
Just checking as there used to be a release version called `dupeGuru PE`, but you are talking about one of the current versions published here at https://github.com/arsenetar/dupeguru/releases.

**Comment by djblanch** [comment]
Thank you, I’ll follow up on that. I am using dupeGuru PE (picture
edition), but I’ll have to check the version. I think it’s from 2012, so
hopefully not the most recent!

Regards,
Dave B.
